linux_install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
Would you mind running the rest of this file through `shellcheck` locally? I can see a few other things that I could suggest manually, but `shellcheck` should catch the majority of them.

Reply:
Needs more double quoting fore sure
```sh
In linux_install.sh line 11:
  cp ff2mpv.json $JSON_DEST
                 ^--------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
  cp ff2mpv.json "$JSON_DEST"


In linux_install.sh line 13:
  mkdir $JSON_DEST
        ^--------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
  mkdir "$JSON_DEST"


In linux_install.sh line 14:
  cp ff2mpv.json $JSON_DEST
                 ^--------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
  cp ff2mpv.json "$JSON_DEST"


In linux_install.sh line 21:
sed -i "s|$OLD_PATH|$CURRENT_DIR/ff2mpv.py|g" $JSON_DEST/ff2mpv.json
                                              ^--------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
sed -i "s|$OLD_PATH|$CURRENT_DIR/ff2mpv.py|g" "$JSON_DEST"/ff2mpv.json

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```